sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/models/DeliveryOutcome.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.models;
+
+import com.azure.core.annotation.Fluent;
+
+/**
+ * There are different outcomes accepted by the AMQP protocol layer.
+ *
+ * Outcomes that don't have any other fields
+ * http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-accepted
+ * http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-released
+ */
+@Fluent
+public class DeliveryOutcome {
```

Comment:
Would there be any reason to create a base `DeilveryOutcome`? If not, should this be an abstract class?

Reply:
You can have a DeliveryOutcome that just has DeliveryState.ACCEPTED or DeliveryState.RELEASED, which has no additional metadata. DeliveryStates like MODIFIED, or REJECTED have metadata, so they extend from this class.